packages/headless/esbuild.mjs

```diff
@@ -128,6 +128,28 @@ const browserUmd = Object.entries(useCaseEntries).map((entry) => {
   );
 });
 
+const quanticUmd = Object.entries(useCaseEntries).map((entry) => {
+  const [useCase, entryPoint] = entry;
+  const outDir = getUseCaseDir('dist/quantic/', useCase);
+  const outfile = `${outDir}/headless.js`;
+
+  const globalName = getUmdGlobalName(useCase);
+
+  return buildBrowserConfig(
+    {
+      entryPoints: [entryPoint],
+      outfile,
+      format: 'cjs',
```

Comment:
Tried the `iife` format, and SF was not able to load the bundle properly. switched to UMD for now and I'm currently investigating the problem of `iife` is SF.

Reply:
It's 'good enough' for a fix, don't necessarily drop the investigation w/ iife, but we can move on and merge this if @erocheleau thinks it's good.
If that's the case, we could have a green release of Quantic tomorrow, which would be swell